### PR TITLE
build using brigade 2

### PR DESCRIPTION
**Pull request by krancour**

### CODE REVIEW

File: .brigade/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+This file was created by brig init and outlines next steps.
```
Comment (vdice) [inline .brigade/NOTES.txt]: Do we want to check this in (here and in the other repos with CI migrating from v1 to v2)?  It looks like this is the stock notes produced from `brig project init` -- maybe the intention was to keep it around for when we would want to add project-specific notes?

**Review by vdice — APPROVED** [review]

File: .brigade/NOTES.txt (hunk 1 above)
Comment (krancour) [reply .brigade/NOTES.txt]: I was iffy on keeping it or not. If you have a preference, I'm happy to go with it.

Comment (vdice) [reply .brigade/NOTES.txt]: Although it doesn't nec. harm anything with it being there, I might have a slight preference towards removing the file if it isn't intended to be updated w/ project-specific notes anytime soon.

Comment (krancour) [reply .brigade/NOTES.txt]: Np. I'm going to amend this PR and the other similar PRs to remove the notes.

**Pull request merged**